cylc/uiserver/authorise.py

```diff
@@ -451,7 +451,6 @@ def auth_failed(self, current_user: str, op_name: str,
                        f":requested to {op_name}.")
         if message:
             log_message = log_message + " " + message
-        Authorization.log.warning(log_message)
```

Comment:
Authorization instances have a `log` attribute, but the class doesn't.

I guess you just wanted to the log the message to the server console?  Maybe not necessary, at `raise web.HTTPError()` (on the next line) seems to do that anyway?

Reply:
This replaces some logging here, which I don't think worked either. IMO this change is fine.